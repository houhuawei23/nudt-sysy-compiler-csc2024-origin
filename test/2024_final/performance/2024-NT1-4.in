42
18373391
