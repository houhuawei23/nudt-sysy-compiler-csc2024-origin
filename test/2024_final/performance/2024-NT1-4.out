-53
65
-119
-73
-95
-27
-55
111
91
20
15
66
34
118
-102
-78
-227
-100
-138
-100
-191
31
23
93
93
32
-20
229
97
216
1
36
7
243
20
138
122
118
-84
167
30
244
0
